db/seeds/base/add_declarations.rb

```diff
@@ -1,106 +1,64 @@
 # frozen_string_literal: true
 
-helpers = Class.new { include ActiveSupport::Testing::TimeHelpers }.new
+if Rails.env.in? %w[development review]
+  helpers            = Class.new { include ActiveSupport::Testing::TimeHelpers }.new
+  lead_provider      = LeadProvider.alphabetical.first # it's very slow to do this for all lead providers
```

Comment:
Can we change it to do this for 2 lead providers - this allows the testers to do API tests and verify changes to one lead providers data does not impact the other